ssl/t1_lib.c

```diff
@@ -3170,10 +3170,11 @@ static int tls_decrypt_ticket(SSL *s, const unsigned char *etick,
     p = sdec;
 
     sess = d2i_SSL_SESSION(NULL, &p, slen);
+    slen -= p - sdec;
```

Comment:
The diff between two pointers will invariably cause a warning on VMS when building with 64-bit pointers, because the result is coerced to a 32-bit value, which *may* lose data.  Not a risk in this case, but still an annoying warning.  We've taken quite a bit of effort in other parts of our source to avoid this...